chunky/src/java/se/llbit/chunky/model/JigsawModel.java

```diff
@@ -68,8 +77,38 @@
   }
 
   private final Quad[] quads;
+  private final Texture[] textures;
 
   public JigsawModel(String orientation) {
+    switch (orientation) {
+      case "up":
+        textures = textureNoLock;
+        orientation = "up_north";
```

Comment:
Why re-assign the orientation here? This is an older world then and the orientation was _not_ up_north in that version.

Reply:
Oh wait, this is the model and not the block. Allright, don't care then :smile: